src/server.ts

```diff
@@ -74,7 +74,7 @@ app.post("/api/find_game", res => {
         const addr: string = Config.useWebSocketDevAddress ? Config.webSocketDevAddress : (Config.webSocketRegions[body?.region] ?? Config.webSocketRegions[Config.defaultRegion]);
 
         res.writeHeader("Content-Type", "application/json");
-        res.end(JSON.stringify({ res: [{ zone: body.zones[0], gameId: "", useHttps: Config.useHttps, hosts: [addr], addrs: [addr] }] }));
```

Comment:
@hsanger is Config.useHttps intended to be null?

Reply:
Nope. Looks like I forgot to change that.